pylint/checkers/base.py

```diff
@@ -1613,7 +1613,7 @@ def _check_self_assigning_variable(self, node):
             if not isinstance(target, nodes.AssignName):
                 continue
             if isinstance(scope, nodes.ClassDef) and target.name in scope_locals:
-                # Check that the scope is different than a class level, which is usually
+                # Check that the scope is different from a class level, which is usually
```

Comment:
Not sure about this change. Doesn't the `if` statement make sue that scope is indeed a class level?

Reply:
Maybe the comment should be moved before the if ?

Reply:
That could be it yeah!